client/app/components/packages/applicant-fieldset.hbs

```diff
@@ -1,23 +1,23 @@
 {{#let @form as |Form|}}
   <Ui::Question
-    @fieldset={{true}}
     class="fieldset relative"
     ...attributes
     as |Q|
   >
-    <Q.Prompt data-test-applicant-title="{{@applicant.friendlyEntityName}}">
+    <Q.Legend data-test-applicant-title="{{@applicant.friendlyEntityName}}">
```

Comment:
"Legend" makes less sense to me. I really liked "Prompt" because it adds clarity to what we're trying to do...